api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -281,7 +285,15 @@ public void setName(String name) {
 	public void setMemberIds(Set<Integer> memberIds) {
 		this.memberIds = new TreeSet<Integer>(memberIds);
 	}
-	
+
+    public Set<Integer> getMembers() {
+        return members;
```

Comment:
Out convention is to create a new instance if members is null that way the method never returns null.